src/config.cpp

```diff
@@ -110,6 +112,7 @@ void parse_program_options(int argc, const char** argv)
              "enable global recording of a raw tracepoint event (usually requires root)")
         ("metric-event,E", po::value(&config.perf_events),
              "the name of a perf event to measure") // TODO: optionally list available events
+        ("clockid,k", po::value(&clock_name), "select timer to use for timestamps")
```

Comment:
I'd change the description to something like:

"clock used for perf timestamps"